feature/bgp/policybase/otg_tests/community_test/README.md

```diff
@@ -0,0 +1,126 @@
+# RT-2.3: BGP Policy Community Set
+
+## Summary
+
+BGP policy configuration for AS Paths and Community Sets
+
+## Subtests
+
+* RT-2.3.1 - Setup BGP sessions
+  * Use cfgLib to generate config for 2 DUT ports.
+  * Use cfgLib to generate config for DUT port 1 eBGP session.
+  * Replace interface and BGP config on DUT.
+
+  * Use helper to configure ATE 2 ports.
+  * Use helper to configure ATE port 1 with 1 eBGP session.
+  * Use helper to configure ATE BGP IPv4 and IPv6 routes.
+  
+  * For IPv4 and IPv6 routes:
+    * Observe received prefixes at ATE port-2.
+  * Generate traffic from ATE port-2 to ATE port-1.
+  * Validate that traffic can be received on ATE port-1 for all installed
+        routes
+
+* RT-2.3.2 - Validate community-set
+  * Configure DUT for each of the following policies
+    * Create a community-set named `my_3_comms` with members and match options as follows
+      * `{ community-member = [ "100:1", "200:2", "300:3" ], match-set-options=ANY }`
+    * Create a community-set named `my_regex_comms` with members and match options as follows
+      * `{ community-member = [ "10[0-9]:1" ], match-set-options=ANY }`
+    * Create a community-set named `all_3_comms` with members and match options as follows
+      * `{ community-member = [ "100:1", "200:2", "300:3" ], match-set-options=ALL }`
+    * Create a community-set named `no_3_comms` with members and match options as follows
+      * `{ community-member = [ "100:1", "200:2", "300:3" ], match-set-options=INVERT }`
+
+  * Configure ATE to
+    * Advertise 2 routes with communities `[100:1, 200:2, 300:3]`
+    * Advertise 2 routes with communities `[100:1, 101:1, 200:1]`
+    * Advertise 2 routes with communities `[110:1]`
+    * Advertise 2 routes with communities `[400:1]`
+  * For each DUT policy configuration
+    * Update the configuration for BGP neighbor policy (`.../apply-policy/config/import-policy`) to the selected community set
+      * Verify prefixes sent, received and installed are as expected
+    * Send traffic
+      * Verify traffic is forwarded for routes with matching policy
+      * Verify traffic is not forwarded for routes without matching policy
+
+* RT-2.3.3 - Validate ext-community-set
+  * Configure DUT for each of the following policies
+    * Create a community-set named `my_3_ext_comms` with members and match options as follows
+      * `{ community-member = [ "100000:100", "200000:200", "300000:300" ], match-set-options=ANY }`
```

Comment:
This are NOT extended communities. It looks like basic community just with 4B ASN.